embedded-bins/runc/Dockerfile

```diff
@@ -2,31 +2,36 @@ ARG BUILDIMAGE
 FROM $BUILDIMAGE AS build
 
 RUN apk add build-base git \
-	curl linux-headers gperf bash pkgconf
+	curl linux-headers gperf bash pkgconf patch cmd:tail
```

Comment:
If we're using patch, I don't think we need git at all here. (otherwise we could use `git apply` instead of `patch -p1`)

Reply:
good idea.